docs/guides/ecosystem/elysia.md

```diff
@@ -0,0 +1,31 @@
+---
+name: Use Elysia
```

Comment:
This title should be more actionable. Assume nobody knows what Elysia is

Maybe something like

> Building type-safe full-stack apps with Elysia & Bun